pyzbar/wrapper.py

```diff
@@ -261,6 +261,26 @@ def zbar_function(fname, restype, *args):
     POINTER(c_int)         # values in ZBarSymbol
 )
 
+zbar_symbol_get_loc_size = zbar_function(
+    'zbar_symbol_get_loc_size',
+    c_uint,
+    POINTER(c_int)         # values in ZBARSymbol
+)
+
+zbar_symbol_get_loc_x = zbar_function(
+    'zbar_symbol_get_loc_x',
+    c_uint,
+    POINTER(c_int),        # values in ZBARSymbol
+    c_uint
+)
+
+zbar_symbol_get_loc_y = zbar_function(
+    'zbar_symbol_get_loc_y',
+    c_uint,
```

Comment:
My copy of `zbar.h` has the return type as `int`, which is represented by `c_int`. Could you check the type in your copy of the header?